packages/web-components/fast-components/src/horizontal-scroll/index.ts

```diff
@@ -32,8 +33,12 @@ export class HorizontalScroll extends FoundationHorizontalScroll {
  * @remarks
  * HTML Element: \<fast-horizontal-scroll\>
  */
-export const fastHorizontalScroll = HorizontalScroll.compose({
+export const fastHorizontalScroll = HorizontalScroll.compose<HorizontalScrollOptions>({
     baseName: "horizontal-scroll",
     template,
     styles,
+    nextFlipper: `<fast-flipper @click=${x =>
+        x.scrollToNext()} aria-hidden="false"></fast-flipper>`,
```

Comment:
Do you think we should set `aria-hidden` to x => x.hiddenFromAT` (I think that does what's expected...)

Reply:
Wouldn't that set it back to the flipper default of true?

Reply:
Good call - we could create an attribute which is slightly different for the Horizontal scroll perhaps and tie to that? Just a thought...

Reply:
Extended the aria-hidden to the horizontal-scroll. It made sense to just pass down the same attribute since only the flippers would use assistive technology.